app/assets/javascripts/discourse/tests/acceptance/bootstrap-mode-notice-test.js

```diff
@@ -34,5 +35,11 @@ acceptance("Bootstrap Mode Notice", function (needs) {
       "/wizard/steps/hello-world",
       "it transitions to the wizard page"
     );
+
+    set(this.siteSettings, "bootstrap_mode_enabled", false)
+    assert.ok(
+      !exists(".bootstrap-mode-notice"),
+      "does not has the bootstrap mode notice"
```

Comment:
Let's try to be more descriptive in the test description

```suggestion
      "removes the bootstrap mode notice when bootstrap mode is disabled"
```

Reply:
sure, done.